controllers/deployment.go

```diff
@@ -64,6 +64,22 @@ func deployment(es *egressv1.ExternalService, configHash string) *appsv1.Deploym
 	a["egress.monzo.com/config-hash"] = configHash
 	a["egress.monzo.com/admin-port"] = strconv.Itoa(int(adPort))
 
+	var resources corev1.ResourceRequirements
+	if es.Spec.Resources != nil {
+		resources = *es.Spec.Resources
+	} else {
+		resources = corev1.ResourceRequirements{
+			Requests: corev1.ResourceList{
+				"cpu":    resource.MustParse("100m"),
+				"memory": resource.MustParse("50Mi"),
+			},
+			Limits: corev1.ResourceList{
+				"cpu":    resource.MustParse("2"),
```

Comment:
Is this default limit a bit high? I guess it doesn't matter as resource allocation is only guaranteed up to requests.

Reply:
I don't think it can hurt